WebRoot/js/zimbraMail/share/view/ZmTagMenu.js

```diff
@@ -194,7 +194,7 @@ function(tagList, addRemove) {
 	var removeid = map ? (map + "_removetag"):this._htmlElId + "|REMOVETAG";
 
 	var miNew = this._menuItems[ZmTagMenu.MENU_ITEM_ADD_ID] = new DwtMenuItem({parent:this, id: addid});
-	miNew.setText(AjxStringUtil.htmlEncode(ZmMsg.newTag));
```

Comment:
usually it is encoded only when we have a placeholder in the message string
but not sure why it was needed here
this is not needed anymore as we have moved encoding to ZmPopupMenu.js